backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/process/adprocess/ADProcessPostProcessService.java

```diff
@@ -372,7 +372,7 @@ else if (ViewOpenTarget.NewBrowserTab.equals(target))
 				return OpenViewAction.builder()
 						.viewId(ViewId.ofViewIdString(viewToOpen.getViewId()))
 						.profileId(ViewProfileId.fromJson(viewToOpen.getProfileId()))
-						.targetTab(recordsToOpen != null ? recordsToOpen.getTargetTab() : RecordsToOpen.TargetTab.SAME_TAB_OVERLAY)
+						.targetTab(recordsToOpen != null ? recordsToOpen.getTargetTab() : RecordsToOpen.TargetTab.NEW_TAB)
```

Comment:
this change made me a bit nervous, but checking the code it seems clear that you are fixing an old bug from commit  https://github.com/metasfresh/metasfresh/commit/b2e0d90701f3a66138bd225aaa13d40ad31779bf 🙏